### PR TITLE
Add opacity option

### PR DESCRIPTION
**Pull request by Hanterdro**
Resolves https://github.com/lukechilds/merge-images/issues/32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,20 @@ mergeImages(['/body.png', '/eyes.png', '/mouth.png'], {
...
+mergeImages([
+  { src: 'body.png', x: 0, y: 0 },
+  { src: 'eyes.png', x: 32, y: 0, opacity: 0.5 }
+])
+  .then(b64 => ...);
+  // data:image/png;base64,iVBORw0KGgoAA...
+```
+
+
 <img src="/test/fixtures/face-custom-dimension.png" width="64">
```
Comment (lukechilds) [inline README.md]: This image is part of the "Dimensions" section above.

File: README.md [hunk 2]
```diff
@@ -78,6 +78,20 @@ mergeImages(['/body.png', '/eyes.png', '/mouth.png'], {
...
+```js
+mergeImages([
+  { src: 'body.png', x: 0, y: 0 },
+  { src: 'eyes.png', x: 32, y: 0, opacity: 0.5 }
+])
+  .then(b64 => ...);
+  // data:image/png;base64,iVBORw0KGgoAA...
+```
+
+
```
Comment (lukechilds) [inline README.md]: Can you add a demo image of what the output will look like, like the other examples?

File: README.md [hunk 3]
```diff
@@ -78,6 +78,20 @@ mergeImages(['/body.png', '/eyes.png', '/mouth.png'], {
 
 Which will look like this:
 
+### Opactiy
```
Comment (lukechilds) [inline README.md]: Typo ;)

File: README.md (hunk 1 above)
Comment (Hanterdro) [reply README.md]: Uhhh copy paste error... didn't saw this in my editor (stupid colors...)

File: README.md (hunk 2 above)
Comment (Hanterdro) [reply README.md]: okay... I will do this tomorrow :)

File: README.md (hunk 3 above)
Comment (Hanterdro) [reply README.md]: fixed

**Comment by Hanterdro** [comment]
Okay... I added an editorconfig file, to force the editor to follow your linting rules.
I didn't add `spaces_around_brackets = inside` because WebStorm ignored this rule (see https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties).

I don't know why, but the fixture I created differs in the unit test from the test result. I created the image in Chrome on macOS and downloaded it. Perhaps you can take a look at this? I don't know how you created the images for the fixtures.
However, on macOS a lot of tests unit tests fail.

File: README.md (hunk 2 above)
Comment (Hanterdro) [reply README.md]: resolved

**Comment by lukechilds** [comment]
>Okay... I added an editorconfig file, to force the editor to follow your linting rules.
I didn't add spaces_around_brackets = inside because WebStorm ignored this rule (see editorconfig/editorconfig/wiki/EditorConfig-Properties).

Thanks but I'd rather not include the `.editorconfig`, you can fix lint errors automatically with `xo --fix`.

>I don't know why, but the fixture I created differs in the unit test from the test result. I created the image in Chrome on macOS and downloaded it. Perhaps you can take a look at this? I don't know how you created the images for the fixtures.
However, on macOS a lot of tests unit tests fail.

I believe this is because the `libpng` library is different on Travis so it generates slightly different output. I've regenerated the images on Linux and now the tests pass.

I should probably have two base64 strings to check against, the Travis version and the macOS version so tests pass locally.

**Pull request merged**